doc/source/parameters/dem/model_parameters.rst

```diff
@@ -1,33 +1,29 @@
 Model Parameters
 -------------------
-In this subsection, DEM simulation parameters are defined. These parameters include ``contact detection method`` and its subsequent information (``dynamic contact search size coefficient`` **or** ``contact detection frequency`` for ``dynamic`` **or** ``constant`` contact detection method), ``repartition method`` (``once``, ``frequent`` or ``dynamic``) for ``load balancing`` (in parallel simulations), ``particle weight`` which defines the particle weight compared to a default ``cell weight`` of 1000, ``neighborhood threshold`` (which defines the contact neighbor list size: ``neighborhood threshold`` * particle diameter), ``particle particle contact force method``, ``particle wall contact force method`` and ``integration method``. 
+In this subsection, contact detection, force models, time integration, load balancing and dynamic contact disabling parameters are defined. 
 
 
 .. code-block:: text
 
  subsection model parameters
-  # Contact detection method
-  subsection contact detection
 
+  subsection contact detection
+    # Contact detection method
     # Choices are constant|dynamic
     set contact detection method                = dynamic
 
-    # Depending on the contact detection method, contact search size coefficient
-    # (safety factor multiplier for dynamic contact search) or contact detection frequency
-    # should be defined for dynamic and constant contact search methods, respectively.
-    set dynamic contact search size coefficient = 0.8
-    set frequency                               = 1
-
     # Particle-particle contact neighborhood size
-    set neighborhood threshold                  = 1
+    set neighborhood threshold                  = 1.3
+
+    set dynamic contact search size coefficient = 0.8
+    set frequency                               = 10
   end
 
-  # Load balancing method and its subsequent information
   subsection load balancing
     # Choices are none|once|frequent|dynamic|dynamic_with_disabling_contacts
     set load balance method                     = none
```

Comment:
```suggestion
    set load balance method                     = once
    set particle weight                         = 10      # Every method, except none
    set step                                    = 1000    # if method = once
    set frequency                               = 1000    # if method = frequent
    set dynamic check frequency                 = 1000    # if method = dynamic
    set threshold                               = 1.3     # if method = dynamic
```

Reply:
This is not necessary. You can be totally against it, but it gives a visual idea to the user of the parameters that could be in this subsection.